snap/snapcraft.yaml

```
@@ -11,7 +11,8 @@ confinement: strict
 
 base: core22
 architectures:
-  - build-on: arm64
+  - build-on: [amd64]
+  - build-on: [arm64]
```

```suggestion
  - build-on: arm64
```
The `amd64` is only used for the tests, so it can be replaced or injected before the amd64 build.